vms/platformvm/client.go

```diff
@@ -235,7 +235,8 @@ type GetSubnetClientResponse struct {
 	Locktime    uint64
 	// subnet transformation tx ID for a permissionless subnet
 	SubnetTransformationTxID ids.ID
-	// subnet manager information for a permissionless L1
+	// subnet conversion information for a permissionless L1
```

Comment:
Are we consistent with the use of "permissioned" referring to traditional Subnets and "permissionless" referring to any converted L1? 

That seems misleading because the manager for the L1 is what determines the permission level for it.

Reply:
These are the only places with this description. Will change to:
```suggestion
	// subnet conversion information for a L1
```

Reply:
`GetSubnetResponse` also has `IsPermissioned` (not new in this PR). I assume it's referring to the same terminology and not permissioned vs elastic.

Fine with deferring it for now though since it's not touched in this PR.